flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -247,6 +265,27 @@ int inferParallelism(FlinkInputFormat format, ScanContext context) {
       parallelism = Math.max(1, parallelism);
       return parallelism;
     }
+
+    boolean localityEnabled() {
+      Boolean localityConfig =
+          this.exposeLocality != null ? this.exposeLocality :
+              readableConfig.get(FlinkConfigOptions.TABLE_EXEC_ICEBERG_EXPOSE_SPLIT_LOCALITY_INFO);
+
+      if (localityConfig != null && !localityConfig) {
+        return false;
+      }
+      FileIO fileIO = table.io();
```

Comment:
Style: this needs a newline above to separate the control flow from the following statement.

Reply:
done